packages/commons/src/tests/e2e/e2eUtils.ts

```diff
@@ -1,19 +1,24 @@
+/* istanbul ignore file */
```

Comment:
The reason why we ignore this file from coverage is the one you mention in the comment at `packages/commons/jest.config.js`? 
If I understand it correctly we are saying that it's not worth it to write unit tests for the files because by running the integration tests we are already "testing" it.

The only minor issue I see is that at the moment we are not enforcing e2e tests on merge, but I see your point and agree on the low value of writing the unit tests for this.

Reply:
If forgot to remove this, the ignore in `jest.config.js` is enough.

Reply:
But yes, that's why I didn't write unit tests for the file.  If I don't ignore it, it'll drop our coverage percentage from 100%.